modules/security-context-constraints-requiring.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: PROCEDURE
+[id="security-context-constraints-requiring_{context}"]
+= Configuring a workload to require a specific SCC
+
+You can configure a workload to require a certain security context constraint (SCC). This is useful in scenarios where you want to pin to a specific SCC to the workload or if you want to prevent your required SCC from being preempted by another SCC in the cluster.
+
+To require a specific SCC, set the `openshift.io/required-scc` annotation on your workload. You can set this annotation on any resource that can set a pod manifest template, such as a deployment or daemon set.
+
+The SCC must already exist in the cluster and must be applicable to the workload, otherwise pod admission will fail. An SCC is considered applicable to the workload if the user creating the pod or the pod's service account has `use` permissions for the SCC in the pod's namespace.
+
+[WARNING]
+====
+Do not change the `openshift.io/required-scc` annotation in the live pod's manifest; the pod admission will fail. To change the required SCC, update the annotation in the underlying pod template, which causes the pod to be deleted and re-created.
+====
+
+.Prerequisites
+
+* The SCC must already exist in the cluster.
+
+.Procedure
+
+. Create a YAML file for the deployment and set a required SCC by setting the `openshift.io/required-scc` annotation:
++
+.Example `deployment.yaml`
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Deployment
+apiVersion: apps/v1
+spec:
+# ...
+  template:
+    metadata:
+      annotations:
+        openshift.io/required-scc: "my-scc" <1>
+# ...
+----
+<1> Specify the name of the SCC to require.
+
+. Create the resource by running the following command:
```

Comment:
Would a `deployment resource` be more specific?

Reply:
I would say no, as the annotation can be used with _any resource that can set a pod manifest template_. Leaving it generic might be better?

Reply:
Yes it can be more than just a deployment.